res/css/photon/message-bar.css

```diff
@@ -26,9 +26,13 @@
   line-height: 1.4;
 }
 
+.multi-line {
+  width: 300px;
```

Comment:
We really don't have to add a class at all :) See my comment below on how to do this.